Makefile

```
@@ -50,11 +53,18 @@ prepare_cpp_project:
 windows-compile:
 	@echo "Compiling..."
 	@$(CASK) $(EMACS) -Q --batch \
-		-l test/windows-bootstrap.el \
+		-l $(WIN-BOOTSTRAP) \
 		-L . -L clients \
 		--eval '(setq byte-compile-error-on-warn t)' \
 		-f batch-byte-compile $(LSP-FILES)
 
+test-pkgs:
```

I would suggest something like to be more clear, but it's your choice
```suggestion
test-downstream-pkgs
```

---

Changed! 👍